src/app/partners/partner/partner.component.html

```diff
@@ -17,4 +17,11 @@
       </mat-card>
     </mat-card>
   </div>
+  <div *ngFor="let contact of contacts" class="belowdiv">
+    Partner Name: {{contact.name}}
```

Comment:
These should be displayed on different lines try adding a <br> in between lines.